### PR TITLE
Reorganize content for easier flow

### PR DESCRIPTION
**Pull request by wviriya**
@vermegi , @anigan , @sikovatc can you please review the PR. I rearranged the content to make it easier to navigate. Better flow for the presenter. Sorry for this last minute, but I will use this new format for Tuesday session.

### CODE REVIEW

File: content/devops/devsecops/2-commit.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Guidance specifically for [Azure pipelines](https://docs.microsoft.com/azure/devops/pipelines/security/overview?view=azure-devops), which describes the different components you can secure.
+
+For workflows hosted on GitHub, there is guidance on [Security hardening](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions). For authenticating your GitHub workflows against Azure Active Directory, there is a (currently preview!) way of authenticating based on [workload identity](https://docs.microsoft.com/azure/active-directory/develop/workload-identity-federation). Guidance on how to configure and use this can be found in the [GitHub docs on Configuring OpenID Connect in Azure](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-azure) and in the [Microsoft docs on Use GitHub Actions to connect to Azure](https://docs.microsoft.com/azure/developer/github/connect-from-azure?tabs=azure-portal%2Cwindows).
+
+![devsecops-controls](./media/devsecops-controls.png)
```
Comment (vermegi) [inline content/devops/devsecops/2-commit.md:51]: All other pages have this one on top. Would prefer to move it to the top here as well.

**Review by vermegi — COMMENTED** [review]
Looks good, just 1 minor remark. 
I like the fact that the content is now closer together per main topic.

File: content/devops/devsecops/2-commit.md (hunk 1 above)
Comment (wviriya) [reply content/devops/devsecops/2-commit.md]: Done

Comment (wviriya) [reply content/devops/devsecops/2-commit.md]: @vermegi can you please approve for merge.

**Review by anigan — APPROVED** [review]

**Review by vermegi — APPROVED** [review]

**Pull request merged**